server/meshmodel/schemas/policy.json

```diff
@@ -0,0 +1,53 @@
+{
+  "$id": "https://schema.meshery.io/policy.json",
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "description": "`Policies` define the genealogy between one or more `Components` interconnected `Components`.",
+  "required": [
+    "apiVersion",
+    "kind",
+    "subType",
+    "model"
+  ],
+  "additionalProperties": false,
+  "type": "object",
+  "properties": {
+    "apiVersion": {
+      "type": "string",
+      "description": "API Version of the Policy."
+    },
+    "metadata": {
+      "type": "object",
+      "description": "Metadata contains additional information associated with the Policy.",
+      "properties": {
+        "description": {
+          "type": "string",
+          "description": "Description of the Policy."
+        },
+        "name": {
+          "$ref": "https://schema.meshery.io/core.json#inputString",
+          "description": "Name of the Policy."
+        },
+        "version": {
```

Comment:
What if the same policy is feasible on multiple version of the same Model, should it be oneOf array or string where it should support wild-card entry as well?

Reply:
Great suggestion.

I too think we should support multiple versions but currently I believe, we don't have that level of sophistication in the backend. Do we want that support to be implemented in this PR too?

Reply:
So the version would be "*" for now? I think that's what should be done, based on our current progress.

Reply:
Updated:

<img width="437" alt="Screenshot 2023-05-31 at 6 56 05 PM" src="https://github.com/meshery/meshery/assets/73700530/b08c3898-2eff-47cd-898a-a1bd7be05854">